docs/installation.rst

```diff
@@ -5,13 +5,13 @@ Installation
 Requirements
 ------------
 Requires:
```

Comment:
I'm not seeing `astropy` on this list, but it should definitely be there.

Reply:
This list/instructions should definitely be updated -- it doesn't actually match the requirements in `setup.cfg`. I think I'll leave that to another PR.